libs/sdk-backend-tiger/src/backend/organization/styling.ts

```diff
@@ -0,0 +1,49 @@
+// (C) 2022 GoodData Corporation
+
+import { MetadataUtilities } from "@gooddata/api-client-tiger";
+import { IOrganizationStylingService } from "@gooddata/sdk-backend-spi";
+import { IThemeMetadataObject, ObjRef } from "@gooddata/sdk-model";
+import { objRefToIdentifier } from "../../utils/api";
+import { convertTheme } from "../../convertors/fromBackend/ThemeConverter";
+import { TigerAuthenticatedCallGuard } from "../../types";
+
+export class OrganizationStylingService implements IOrganizationStylingService {
+    constructor(public readonly authCall: TigerAuthenticatedCallGuard) {}
+
+    public async getThemes(): Promise<IThemeMetadataObject[]> {
+        return await this.authCall(async (client) => {
+            return MetadataUtilities.getAllPagesOf(client, client.entities.getAllEntitiesThemes, {})
+                .then(MetadataUtilities.mergeEntitiesResults)
+                .then((themes) => themes.data.map(convertTheme));
+        });
+    }
+
+    public async setActiveTheme(themeRef: ObjRef): Promise<void> {
+        const themeId = await objRefToIdentifier(themeRef, this.authCall);
+
+        // It is not possible to PUT activeTheme if it does not exist already,
+        // therefore we first try to DELETE it and POST a new one
+        try {
+            await this.authCall(
+                async (client) =>
+                    await client.entities.deleteEntityOrganizationSettings({
```

Comment:
nitpick: async/await in nested call is redundant - can be simplified to
```
(client) => client.entities.deleteEntityOrganizationSettings({
```
same applies the authCall in finally statement

Reply:
Updated, thank you